submissions/benjaminKamena/Dom/product.js

```diff
@@ -0,0 +1,54 @@
+// product validation
+let quantity = document.getElementById("quantity"),
+    price = document.getElementById("price"),
+    shippingCost = document.getElementById("shippingCost"),
+    total = document.getElementById("totalCost"),
+    submit = document.getElementById("btn"),
+    perUnitPrice = price.innerHTML,
+    productPrice = perUnitPrice * quantity.value;
+
+function quantityField(){
+    let productRegex = /^[1-9]+[0-9]*$/;
+    if(!quantity.value.match(productRegex)){
+        submit.disabled = true;
+        quantity.nextElementSibling.innerHTML = "Please enter a valid quantity";
+    } else{
+        submit.disabled = false;
+        quantity.nextElementSibling.innerHTML = "";
+    }
+}
+
+function cost(){
+    shippingAmount();
+    if(quantity.value > 1){
+        productPrice = perUnitPrice * quantity.value;
+        totalCost.innerHTML = productPrice + shipping;
```

Comment:
I think you should place this `shippingAmount();` inside the `if`  block. Cos you want to even calculate for shipping when quantity is greater than 1

Reply:
Also instead of doing `productPrice` calculation inside the `shippingAmount()`, pass it as an argument from here to be used as a parameter in the function

Reply:
You haven't done what I said in the second statement